kibana-reports/public/application.tsx

```diff
@@ -20,16 +20,15 @@ import { AppPluginStartDependencies } from './types';
 import { OpendistroKibanaReportsApp } from './components/app';
 
 export const renderApp = (
-  { notifications, http, chrome }: CoreStart,
-  { navigation }: AppPluginStartDependencies,
+  { http, chrome, uiSettings }: CoreStart,
```

Comment:
If the `uiSettings` parameter is used to refer specifically to `commonlyUsedRanges`, can we use a more descriptive name for it? `uiSettings` is very generic

Reply:
uiSettings is the name of the service, I see no reason for not using the default name.

Reply:
It's the exact name used across kibana source code.

Reply:
I see, no issue then